go/chat/flipmanager.go

```diff
@@ -544,7 +542,11 @@ func (m *FlipManager) StartFlip(ctx context.Context, uid gregor1.UID, hostConvID
 	if err != nil {
 		return err
 	}
-	if err := m.G().ChatHelper.SendTextByID(ctx, conv.GetConvID(), tlfName, string(infoBody)); err != nil {
+	if err := m.G().ChatHelper.SendMsgByID(ctx, conv.GetConvID(), tlfName,
+		chat1.NewMessageBodyWithFlip(chat1.MessageFlip{
+			Text:   string(infoBody),
+			GameID: gameID,
```

Comment:
drop `gameID` from `hostMessageInfo` since it's part of the message now?

Reply:
Hmm, yeah good point.

Reply:
Actually I'm going to keep it there and double check messages against it.